### PR TITLE
allow users to create and join teams

### PR DESCRIPTION
**Pull request by dkhgh**

### CODE REVIEW

File: app/models/team.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class Team < ActiveRecord::Base
+  belongs_to :owner, class_name: "User", foreign_key: "user_id", touch: true
+  has_many :memberships, :dependent => :destroy
```
Comment (fatboypunk) [inline app/models/team.rb]: ruby 2 syntax :)

**Comment by dkhgh** [comment]
@fatboypunk should a team owner also be a member?

**Comment by fatboypunk** [comment]
He already is right?

**Comment by fatboypunk** [comment]
Don't think you have to add it as a membership

**Comment by dkhgh** [comment]
He's related but not through 'memberships'

**Pull request merged**